src/transformState.js

```diff
@@ -1,11 +1,32 @@
-'use strict';
+"use strict";
 
 /**
  * @param {Object} state
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    switch (obj.type) {
+      case "addProperties":
+        for (const keys in obj.extraData) {
+          state[keys] = obj.extraData[keys];
```

Comment:
use `Object.assing` here for it

Reply:
@SerhiiKirik I am not sure that I can use this method. 
The task is to modify "state" according to the type, if i use Object.assing, it means that I create a new object.
but in this part of the code i have to add extra object to existing object 'state'

Reply:
i corrected it